### ISSUE TITLE
Accessibility - Projects page - form element missing label

### ISSUE THREAD

**Issue by terrimorton** [dotnet-foundation/website#864]
This issue was found by running the Accessibility Insights for Web browser extension in Chrome.

URL: https://dotnetfoundation.org/projects

![image](https://user-images.githubusercontent.com/4643903/118022237-22df7400-b32a-11eb-882d-c2d2b859d30e.png)

Snippet:

- `<input class="form-control" id="textToSearch" name="textToSearch" type="search" value="" placeholder="">`

Fix ONE of the following:

- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element has an empty placeholder attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by terrimorton** [comment]
Thank you for your contributions to fix these accessibility issues, @Arhell!
